FIRO_TSEnsembles/src/main/java/hec/ensemble/ui/EnsemblePicker.java

```diff
@@ -37,4 +37,12 @@ public void tableChanged(ListSelectionEvent e) {
      public int getSelectedRow() {
          return selectedRow;
      }
+
+     public String getSelectedPath() {
+         int row = table.getSelectedRow();
+         String location = table.getModel().getValueAt(row, 0).toString();
+         String parameter = table.getModel().getValueAt(row, 1).toString();
+
+         return location+"/"+parameter;
+     }
```

Comment:
getSelectedPath() needs more work/design.   Do we require the calling application to split on '/' and construct a TimeSeriesIdentifier?